package-lock.json

```diff
@@ -1,13 +1,4803 @@
 {
   "name": "cloud-foundry-config-client",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@babel/code-frame": {
+      "version": "7.12.13",
+      "resolved": "https://artifactory.platform.manulife.io/artifactory/api/npm/institutional-grs-npm/@babel/code-frame/-/code-frame-7.12.13.tgz",
```

Comment:
You should remove this lock file (if it existed before, that was a mistake that I missed from a previous PR 😬). This project uses `yarn` and should reference a public registry.

Reply:
Deleted